interface/ceed-python/libceed/ceed_constants.py

```diff
@@ -94,6 +94,7 @@ def __init__(self, constant):
 # Vectors
 VECTOR_ACTIVE = _CeedConstantObject(lib.CEED_VECTOR_ACTIVE)
 VECTOR_NONE = _CeedConstantObject(lib.CEED_VECTOR_NONE)
+VECTOR_NULL = _CeedConstantObject(ffi.NULL)
```

Comment:
What about a more general purpose `libceed.NULL`? We use null arguments in a few places.

Reply:
Keyword argument defaulting to `None`. Let's not define our own thing.